modules/core/src/main/java/org/apache/ignite/spi/metric/jmx/ReadOnlyDynamicMBean.java

```diff
@@ -57,16 +58,19 @@ else if ("invoke".equals(actionName))
 
     /** {@inheritDoc} */
     @Override public AttributeList getAttributes(String[] attributes) {
-        AttributeList list = new AttributeList();
-
+        AttributeList attributeList = new AttributeList();
+        List<Attribute> list = attributeList.asList();
         try {
             for (String attribute : attributes) {
                 Object val = getAttribute(attribute);
-
-                list.add(val);
+                if( val instanceof Attribute ) {
```

Comment:
Following https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-BracesandIndentation this code should look like as follows:
```
            for (String attribute : attributes) {
                Object val = getAttribute(attribute);

                if (val instanceof Attribute)
                    list.add((Attribute) val);
                else
                    list.add(new Attribute(attribute, val));
            }
```